tomviz/PipelineView.cxx

```diff
@@ -303,6 +303,14 @@ void PipelineView::contextMenuEvent(QContextMenuEvent* e)
     deleteAction->setEnabled(false);
   }
 
+  // Disable deletion for "Output" data sources
+  if (dataSource != nullptr) {
+    auto output = dataSource->property("output");
+    if (output.isValid() && output.toBool()) {
+      deleteAction->setEnabled(false);
+    }
+  }
+
```

Comment:
Shouldn't you just modify the enableDeleteItems() function?

Reply:
Probably, thanks for the tip :smile_cat: